packages/lib/server/username.ts

```diff
@@ -7,7 +6,7 @@ import prisma from "@calcom/prisma";
 import { IS_CALCOM } from "../constants";
 import notEmpty from "../notEmpty";
 
-const WORDLIST_PATH = process.env.PREMIUM_WORDLIST_PATH_FROM_SERVER ?? "";
+const cachedData: Set<string> = new Set();
```

Comment:
Sets are way more efficient than arrays and less ugly than key/value objects with 1'd values. Note this is stored in lambda cache so provides a little perf boost (saves the network call)